qemu-plugin-sys/generate-bindings.rs

```diff
@@ -1,13 +1,17 @@
-#!/usr/bin/env -S cargo +nightly -Z script
+#!/usr/bin/env -S cargo +nightly-gnu -Z script
 
 //! ```cargo
+//! [package]
+//! edition = "2021"
 //! [dependencies]
 //! anyhow = "*"
 //! bindgen = "*"
 //! cargo_metadata = "*"
 //! reqwest = { version = "*", features = ["blocking"] }
 //! tar = "*"
 //! xz2 = "*"
+//![lints.rust]
+//!non_snake_case = "allow"
```

Comment:
again, to suppress warnings. It seems that this generate-bindings.rs script is intended to run standalone, but it compiles everything under src/ directory anyway. I wonder if there's a way to stop it doing that.